workspaces/orchestrator/README.md

```diff
@@ -1,115 +1,53 @@
 # Orchestrator Plugin for Backstage
 
-The Orchestrator for Backstage is a mechanism designed to facilitate the implementation and execution of developer self-service flows. It serves as a vital component that enhances and augments the existing scaffolder functionality of Backstage with a more flexible and powerful set of features including long-running and asynchronous flows.
+The Orchestrator for Backstage is designed to enable self-service flows. It serves as a vital component that enhances and augments the existing scaffolder functionality of Backstage with a more flexible and powerful set of features including long-running and asynchronous flows.
 
-The orchestrator works harmoniously with other Backstage components such as the Software Catalog, permissions, and plugins as well as others. By leveraging its capabilities, organizations can orchestrate and coordinate developer self-service flows effectively.
-
-## Context
```

Comment:
why removing the `Context` section? I see that you're keeping some of the info here and there but I don't get why the removed info is unnecessary. Need clarifications here :)

Reply:
I found it crowds the README and is related to Sonata itself and not to the plugin
But I can return it

Reply:
partially reverted